github-issues.go

```diff
@@ -63,9 +63,6 @@ type RepoIssues struct {
 		Color   string `json:"color"`
 		Default bool   `json:"default"`
 	} `json:"labels"`
-	Assignee struct {
```

Comment:
I checked one more time. 
Now, I see there is a single reference to it in index.html file that seems to be I missed when I made the change. I'll revert this one back.